orb/core/connection_types/sql/sqlite/statements/select.py

```diff
@@ -34,7 +34,7 @@ def __call__(self, model, context, fields=None):
 
         # process columns to select
         for column in sorted(columns, self.cmpcol):
-            if column.testFlag(column.Flags.Virtual):
+            if column.testFlag(column.Flags.Virtual) and not issubclass(model, orb.View):
```

Comment:
note to self: need to figure out a better way to group some of the logic & separate the SQL queries between the different backends